Problem4/test/problem4/Problem4Test.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package problem4;
+
+import java.util.Random;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Evan
+ */
+public class Problem4Test {
+    
+    private static int[] numbers;
+    
+    @Before
+    public void setUp()
+    {
+        numbers = new int[10]; //Create an array of integers
+        numbers[0] = 88;
+        numbers[1] = 32;
+        numbers[2] = 91;
+        numbers[3] = 13;
+        numbers[4] = 35;
+        numbers[5] = 100;
+        numbers[6] = 1;
+        numbers[7] = 24;
+        numbers[8] = 12;
+        numbers[9] = 73;
+    }
+
+    /**
+     * Test of swapSort method, of class Problem4.
+     */
+    @Test
+    public void testSwapSort() {
```

Comment:
Your testing should include edge cases like empty arrays and arrays of a single element.